settings.py

```diff
@@ -15,10 +15,15 @@ def read_bool(value: str) -> bool:
 
 CODE_HOTRELOAD = read_bool(os.environ["CODE_HOTRELOAD"])
 
-AWS_REGION = os.environ["AWS_REGION"]
-AWS_ACCESS_KEY_ID = os.environ["AWS_ACCESS_KEY_ID"]
-AWS_SECRET_ACCESS_KEY = os.environ["AWS_SECRET_ACCESS_KEY"]
-AWS_ENDPOINT_URL = os.environ["AWS_ENDPOINT_URL"]
-AWS_BUCKET_NAME = os.environ["AWS_BUCKET_NAME"]
+S3_AWS_REGION = os.environ["AWS_REGION"]
+S3_AWS_ACCESS_KEY_ID = os.environ["AWS_ACCESS_KEY_ID"]
+S3_AWS_SECRET_ACCESS_KEY = os.environ["AWS_SECRET_ACCESS_KEY"]
+S3_AWS_ENDPOINT_URL = os.environ["AWS_ENDPOINT_URL"]
+S3_AWS_BUCKET_NAME = os.environ["AWS_BUCKET_NAME"]
+
+REKOGNITION_AWS_REGION = os.environ["REKOGNITION_AWS_REGION"]
+REKOGNITION_AWS_ACCESS_KEY_ID = os.environ["REKOGNITION_AWS_ACCESS_KEY_ID"]
+REKOGNITION_AWS_SECRET_ACCESS_KEY = os.environ["REKOGNITION_AWS_SECRET_ACCESS_KEY"]
+REKOGNITION_AWS_ENDPOINT_URL = os.environ["REKOGNITION_AWS_ENDPOINT_URL"]
```

Comment:
Still needs additions in vault before rollout

Reply:
Also, I know aws services have a commonality of often depending on env variables by special names, which we previously used:

```py
AWS_REGION = os.environ["AWS_REGION"]
AWS_ACCESS_KEY_ID = os.environ["AWS_ACCESS_KEY_ID"]
AWS_SECRET_ACCESS_KEY = os.environ["AWS_SECRET_ACCESS_KEY"]
AWS_ENDPOINT_URL = os.environ["AWS_ENDPOINT_URL"]
AWS_BUCKET_NAME = os.environ["AWS_BUCKET_NAME"]
```

We will want to be careful during the rollout to identify any cases where we were previously depending on this behaviour -- there may be breaking changes.

Reply:
All of the new env vars have been added. The removed ones will stay until after the safe deployment.

Reply:
> We will want to be careful during the rollout to identify any cases where we were previously depending on this behaviour -- there may be breaking changes.

On second thought this is not a concern as we will not use these APIs to begin with. We can safely remove the old vars before anything will call this.